tests/unit_tests/all/plugins/test_plugin_manager.py

```diff
@@ -17,7 +17,7 @@ def test_no_plugins():
     assert pm.get_ecl100_config_path() is None
     assert pm.get_ecl300_config_path() is None
 
-    assert len(pm.get_installable_jobs()) > 0
```

Comment:
Some tests still test this behavior via INSTALLABLE_JOBS by getting some mocked stuff from `dummy_plugins.py`, ref `test_with_plugins` and `test_job_documentation`. Furthermore if it were to not work it is tested indirectly via integration tests / other tests that run the forward models in ERT would break.